src/main/resources/changelogs/0003-add-unique-constraint-to-cart.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="1_delete_duplicate_carts" author="raluca">
+        <sqlFile encoding="utf8" path="sql/delete-duplicate-carts-script.sql"/>
+    </changeSet>
+
+    <changeSet id="2_add_uq_constr_to_cart_on_user" author="raluca">
+        <preConditions>
+            <sqlCheck expectedResult="0">
+                select count(*)
+                from
+                information_schema.table_constraints
+                where table_name='cart'and
+                table_schema='vinylstore' and
+                constraint_type='UNIQUE';
+            </sqlCheck>
+        </preConditions>
+        <addUniqueConstraint tableName="cart" columnNames="user_id"
+                             schemaName="vinylstore" constraintName="uq_user_cart"/>
```

Comment:
we used the convention [constraint_type]_[table]_[column_name] for the FK constraints, lets use that here as well:
uq_cart_user_id

Reply:
done

Reply:
i get this error when i do that:
->>> changelogs/0003-add-unique-constraint-to-cart.xml::2_add_uq_constr_to_cart_on_user::raluca was: 8:01d72392fb260518730da0826dca0ab5 but is now: 8:0c66d743bc2b5f6a37fb23e8c461f41c

Reply:
because you already ran the liquibase scripts :) cleanup databasechangelog